config/plugins/fuses.md

```diff
@@ -0,0 +1,60 @@
+---
+description: Toggle Electron functionality at package-time with Electron Fuses.
+---
+
+> Added in version 6.1.0
+
+# Fuses Plugin
+
+This plugin allows flipping [Electron Fuses](https://github.com/electron/fuses) when packaging your app with Electron Forge. Fuses are bits in the Electron binary that allow certain features to be enabled/disabled when your app is packaged.
```

Comment:
I would explain a bit more in the intro paragraph what the purpose of fuses are (recap some of https://www.electronjs.org/docs/latest/tutorial/fuses and link back to the OG doc).